src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_3_layout.html.twig

```diff
@@ -130,14 +130,14 @@
 {% block form_row -%}
     {%- set widget_attr = {} -%}
     {%- if help is not empty -%}
-        {%- set widget_attr = {attr: {'aria-describedby': id ~"_help"}} -%}
+        {%- set widget_attr = {attr: {'aria-describedby': id ~ '_help'}} -%}
     {%- endif -%}
     <div{% with {attr: row_attr|merge({class: (row_attr.class|default('') ~ ' form-group' ~ ((not compound or force_error|default(false)) and not valid ? ' has-error'))|trim})} %}{{ block('attributes') }}{% endwith %}>
-        {{- form_label(form) }} {# -#}
-        {{ form_widget(form, widget_attr) }} {# -#}
+        {{- form_label(form) }} {#-#}
+        {{ form_widget(form, widget_attr) }} {#-#}
```

Comment:
I think `{# -#}` is more readable. With `{#-#}` you have to know which side wins. 
So I would use ``{# -#}`, `{#- #}` and `{#--#}`.